processes/util/google_integration.py

```diff
@@ -0,0 +1,44 @@
+from io import BytesIO
+import json
+import os
+from googleapiclient.discovery import build
+from googleapiclient.http import MediaIoBaseDownload
+from google.oauth2.service_account import Credentials
+
+SERVICE_ACCOUNT_FILE = os.environ['GOOGLE_SERVICE_ACCOUNT_FILE']
+
+from logger import create_log
+
+logger = create_log(__name__)
+
+def create_drive_service(service_account_info):
+    scopes = ['https://www.googleapis.com/auth/drive']
+    credentials = Credentials.from_service_account_info(service_account_info, scopes=scopes)
+
+    return build('drive', 'v3', credentials=credentials)
+
+def get_drive_file(file_id: str) -> BytesIO:
+    drive_service = create_drive_service(service_account_info=json.loads(SERVICE_ACCOUNT_FILE))
```

Comment:
I like the use of functions rather than a class here. Could we create the drive_service at the top level of this file so it's used more like a singleton?